lection3/task2.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+// 1) x2 - 6x + 9 = 0
+// 2) x2 - 4x - 5 = 0
+
+function calculate(a, b, c) {
+    let radical1, radical2, discriminant, answer;
+    discriminant = b*b - (4*a*c);
+    radical1 = (- b - Math.sqrt(discriminant))/2;
+    radical2 = (- b + Math.sqrt(discriminant))/2;
+    if (radical1 === radical2){answer = radical1} 
```

Comment:
Add "prettier" extention) and place the body of "if" statement on the next line

Reply:
done